applications/nrf5340_audio/src/audio/audio_sync_timer.c

```diff
@@ -79,6 +84,78 @@ static void event_handler(nrf_timer_event_t event_type, void *ctx)
 {
 }
 
+void sync_led_1_on()
+{
+	nrfx_timer_compare(&timer_instance, NRF_TIMER_CC_CHANNEL3, audio_sync_timer_curr_time_get(),
+			   true);
+	nrfx_gpiote_out_task_trigger(BLINKLED);
+}
+
+void sync_led_1_compare_time_set_update(uint32_t set_time_us)
+{
+	nrfx_timer_compare(&timer_instance, NRF_TIMER_CC_CHANNEL2, set_time_us, true);
```

Comment:
What is the reason for choosing CC channel 2 and 3?

Reply:
When using CC channel 0 and 1 or channel 1 and 2 the functionality does not work. It seems that those might be used elsewhere?

Reply:
The sync timer (timer 1) should only be used from this file. Try to check the rest of the code in audio_sync_timer.c to see if you find out if/where channel 0 and 1 are used.

Reply:
Currently channel 0 and 1 is used as a task for capturing the timer value for I2S framestart and current time. Is it possible that when using a capture task on a given channel it is on the same channel as the CC channels.

Reply:
After reading the documentation I believe I have found the reason why the functionality doesn't work when using CC channel 0 and 1.

Reply:
After reading the documentation for the Timer Peripheral I believe I have found the reason the functionality doesn't work when using CC channels 0 and 1.
Currently the functions for getting current time stamp and time stamp when i2s started are using the capture task on CC channel 0 and 1. 
And every time these functions are called upon (which are frequently) the counter value is copied to the CC[n] register, which is the same register that is the compare event uses. 
Therefor when setting a compare time on channel 0 or 1, this value gets overwritten when the other functions get called upon.

Reply:
![image](https://user-images.githubusercontent.com/60898006/181206748-4b131121-7fc0-48a9-a309-57bb861eff3c.png)